commit eeb930172c353b4649f88a35feaaeb0fc354e47b
Author: Ulrich Drepper <drepper@redhat.com>
Date:   Wed May 14 22:52:44 2008 +0000

    (BODY for __gconv_transform_ascii_internal): Add missing braces.
    (BODY for __gconv_transform_internal_ascii): Likewise.

---
 iconv/gconv_simple.c |   18 +++++++++++-------
 1 file changed, 11 insertions(+), 7 deletions(-)

--- a/iconv/gconv_simple.c
+++ b/iconv/gconv_simple.c
@@ -1,5 +1,5 @@
 /* Simple transformations functions.
-   Copyright (C) 1997-2003, 2004, 2005, 2007 Free Software Foundation, Inc.
+   Copyright (C) 1997-2005, 2007, 2008 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1997.
 
@@ -820,9 +820,11 @@
 	STANDARD_FROM_LOOP_ERR_HANDLER (1);				      \
       }									      \
     else								      \
-      /* It's an one byte sequence.  */					      \
-      *((uint32_t *) outptr) = *inptr++;				      \
-      outptr += sizeof (uint32_t);					      \
+      {									      \
+	/* It's an one byte sequence.  */				      \
+	*((uint32_t *) outptr) = *inptr++;				      \
+	outptr += sizeof (uint32_t);					      \
+      }									      \
   }
 #define LOOP_NEED_FLAGS
 #include <iconv/loop.c>
@@ -851,9 +853,11 @@
 	STANDARD_TO_LOOP_ERR_HANDLER (4);				      \
       }									      \
     else								      \
-      /* It's an one byte sequence.  */					      \
-      *outptr++ = *((const uint32_t *) inptr);				      \
-      inptr += sizeof (uint32_t);					      \
+      {									      \
+	/* It's an one byte sequence.  */				      \
+	*outptr++ = *((const uint32_t *) inptr);			      \
+	inptr += sizeof (uint32_t);					      \
+      }									      \
   }
 #define LOOP_NEED_FLAGS
 #include <iconv/loop.c>
